### PR TITLE
Use StrEnum for car classes

### PR DESCRIPTION
**Pull request by jnewb1**
useful for auto fingerprinting, to get the python names of the cars

### CODE REVIEW

File: selfdrive/car/chrysler/values.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # ruff: noqa: E501
```
Comment (adeebshihadeh) [inline selfdrive/car/chrysler/values.py:1]: side note: can we remove this noqa?

**Review by adeebshihadeh — APPROVED** [review]

File: selfdrive/car/chrysler/values.py (hunk 1 above)
Comment (jnewb1) [reply selfdrive/car/chrysler/values.py:1]: fingerprints are long, especially the CAN ones. Once the fingerprints are in their own file, we can remove them

**Comment by sshane** [comment]
@jnewb1 pytest says it can't serialize CAR enum when running with multiple jobs, fine if you don't specify the -n flag

**Comment by jnewb1** [comment]
> @jnewb1 pytest says it can't serialize CAR enum when running with multiple jobs, fine if you don't specify the -n flag

hmm, which test / environment? we run with multiple cores in CI

**Comment by sshane** [comment]
Oh, it seems it's only an issue when something fails inside a subtest where the enum instance is used (test_toyota for example)

**Pull request merged**